Source/WTF/wtf/spi/darwin/CommonCryptoSPI.h

```diff
@@ -34,6 +34,11 @@
 #endif
 #endif
 
+#if !PLATFORM(IOS) || USE(APPLE_INTERNAL_SDK)
```

Comment:
I thought about that but decided to guard it just in case ;) I will remove it in a second